### ISSUE TITLE
Select Effects doesn't work as expected

### ISSUE THREAD

**Issue by KX1095** [DrozmotiX/ioBroker.wled#490]
**Describe the bug**  
I want to select effects via the adapter, but nothing happening. I really need this functionality to improve my automation.
I tried it over the iobroker.javascript adapter ( `setState('wled.{instance}.{clientId}.seg.{segmentIndex}.fx', {effectIndex | effectName})` ), also in the objects overview of iobroker. It this feature not implemented yet?

**To Reproduce**  
1. Go to 'Objects'
2. Click on 'wled' > instance (in my case 0) > deviceId > 'seg' > any segment (e.g. 0)
3. Scroll down to 'fx'
4. Click on the current value in the value column
5. Select a effect from the select-field in the popup
6. Click set value
7. bug -> no changes in wled or on the leds

**Expected behavior**  
I want to set effects via the iobroker.javascript adapter with schedule commands as i described earlier. The problem is if I change colors for my 3 segments in a interval of 10 seconds to imitate a "Blink" or "Breath" effect, when the configured segments will disappear after some time on the ESP and I must reconfigured they.

**Versions:**  
 - Adapter version: 0.6.7
 - JS-Controller version: 4.0.24
 - Node version: 16.18.1
 - Operating system: Raspbian Buster

**Additional context**  
In my opinion I described the problem good enough. If you need some contribution with this problem I will do my best.

**Comment by DutchmanNL** [comment]
Which WLED firmware version are u using ?

**Comment by KX1095** [comment]
> Which WLED firmware version are u using ?

I use WLED version 0.13.3 "Toki", build 2208222 on my ESP's.

**Comment by DutchmanNL** [comment]
> > Which WLED firmware version are u using ?
> 
> I use WLED version 0.13.3 "Toki", build 2208222 on my ESP's.

hmm i will check this weekend, as far I remember I had no issue with that firmware version and related effects.
Is there any error message shown in the log when sending an effect change ?

**Comment by KX1095** [comment]
I had tested the behavior moments ago to produce eventually fresh error loggins, and now it works. 🤔

Maybe my ESP WLED firmware was needed a restart which I did earlier today. My problem was not all segments changed her color correctly, and a change of one segment color over your adapter changed the color of another segment. I also had tried to reinstall the adapter in iobroker but it did not help. So I did restarted the problematic ESP.
